src/types/utility.types.ts

```diff
@@ -1,4 +1,6 @@
 import { SxProps } from '@mui/material';
-import { theme } from '../theme';
+import { createTheme } from '@mui/material/styles';
+import { themeTokens } from '../theme/themeTokens';
 
+const theme = createTheme(themeTokens);
 export type MuiStylesObject = Record<string, SxProps<typeof theme>>;
```

Comment:
Why don't you use here just a `Theme` type from `mui`? It is a rare and unpreferable case when there is a need to create a value to extract type.